tdrs-backend/tdpservice/users/test/test_auth.py

```diff
@@ -83,6 +84,22 @@ def test_oidc_logout_with_token(api_client):
     response = view(request)
     assert response.status_code == status.HTTP_302_FOUND
 
+@pytest.mark.django_db
```

Comment:
This test ensures the cookie is updated with a new timestamp when a request is made to the API